src/main/resources/aws/2-nodes-dc.yaml

```diff
@@ -127,75 +123,57 @@ Resources:
     Properties:
       VpcId:
         Ref: Vpc
-      GroupDescription: Enables SSH access
+      GroupDescription: For SSH enabled instances
       SecurityGroupIngress:
         -
           IpProtocol: tcp
           FromPort: 22
           ToPort: 22
           CidrIp: 0.0.0.0/0
```

Comment:
@dagguh I recalled that during our video call in the past you suggested that SSH access should also be granted by `AccessProvider` similarly to other port access. That would be to avoid complexity of using 2 different mechanisms.

Reply:
I didn't implement this suggestion and I'm thinking if I could skip it, because there is pressure to merge this PR

Reply:
Yep, the scope is quite big. Raised https://ecosystem.atlassian.net/browse/JPERF-763